src/cup1d/data/data_fid_eBOSS_mock.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+import pandas
+import numpy as np
+
+from cup1d.data.base_p1d_data import BaseDataP1D, _drop_zbins
+
+
+class P1D_eBOSS_mock(BaseDataP1D):
+    def __init__(
+        self,
+        diag_cov=False,
+        kmax_kms=0.09,
+        zmin=2.19,
+        zmax=None,
+    ):
```

Comment:
I would also add here an option to specify the Nyx simulation used, in case we have more eBOSS mocks from other Nyx sims (not just the fiducial). 

Then we could rename the module `data_eBOSS_mock.py` without specifying `_fid_` in the name.

Reply:
done